ClientHints.php

```diff
@@ -217,6 +217,10 @@ public static function factory(array $headers): ClientHints
         $fullVersionList = [];
 
         foreach ($headers as $name => $value) {
+            if (\is_string($value)) {
+                $value = \str_replace('"', '', $value);
```

Comment:
this will actually break the detection of `sec-ch-ua-full-version-list`, as it contains a coulple of `"` on purpose. We maybe should do something like this for each expected string value:
```php
$x = preg_replace('/^"(.*)"$/', '$1', $value);
```

Reply:
or str_replace only for platformversion, platform and uafullversion

Reply:
yes. But I'm not sure if there might be cases in the future where a `"` is contained within as well. But guess we can also handle it if it makes problems later.